docs/Reference/JSON-RPC-API-Methods.md

```diff
@@ -2191,4 +2191,193 @@ Proposes [adding or removing a validator](../Consensus-Protocols/IBFT.md#adding-
 
 ## Permissioning Methods
 
-Permissioning is under development and will be available in v1.0. 
\ No newline at end of file
+!!! note 
+    Permissioning is under development and will be available in v1.0. 
+
+!!! note
+    The `PERM` API methods are not enabled by default. Use the [`--rpc-http-api`](Pantheon-CLI-Syntax.md#rpc-http-api) 
+    or [`--rpc-ws-api`](Pantheon-CLI-Syntax.md#rpc-ws-api) options to enable the `PERM` API methods.
+
+### perm_addAccountsToWhitelist
+
+Adds accounts (participants) to the accounts whitelist. 
+
+**Parameters** 
+
+`list of strings` - List of account addresses 
+
+**Returns** 
+
+`result` - `Success` or `error`
+
+!!! example
+    ```bash tab="curl HTTP request"
+    $ curl -X POST --data '{"jsonrpc":"2.0","method":"perm_addAccountsToWhitelist","params":[["0xb9b81ee349c3807e46bc71aa2632203c5b462032", "0xb9b81ee349c3807e46bc71aa2632203c5b462034"]], "id":1}' <JSON-RPC-http-endpoint:port>
+    ```
+    
+    ```bash tab="wscat WS request"
+    {"jsonrpc":"2.0","method":"perm_addAccountsToWhitelist","params":[["0xb9b81ee349c3807e46bc71aa2632203c5b462032", "0xb9b81ee349c3807e46bc71aa2632203c5b462034"]], "id":1}
```

Comment:
Explain why we need double square brackets because it's not obvious, I first thought it was a typo, but then tested and understood that the params list contains a param that is a list even if this param is the only one in the data list. If we miss the list in the list, it returns a message like `{"jsonrpc":"2.0","id":1,"error":{"code":-32603,"message":"Internal error"}}`

Reply:
Added an explanation. You're right - this caught me the first time as well.